app/scripts/lib/createRPCMethodTrackingMiddleware.js

```diff
@@ -0,0 +1,85 @@
+import { SECOND } from '../../../shared/constants/time';
+
+const USER_PROMPTED_EVENT_NAME_MAP = {
+  eth_signTypedData_v4: 'Signature Requested',
```

Comment:
Super minor but can we make these strings a constant so that we don't accidentally misspell one?

Reply:
Resolved in latest commit :)